### DISCUSSION TITLE
Tap maintenance and Homebrew development

### DISCUSSION THREAD

**Discussion by masaccio** [Homebrew/discussions#5027] [Tap maintenance and Homebrew development] [answered]
### Output of `brew config`

```shell
-
```

### Output of `brew doctor`

```shell
-
```

### Description of issue

What’s the procedure for adding a new PyPI package? Is it as straightforward as a PR changing pypi_formula_mappings.json to get picked up a build process? I’ve not spotted any description of this so looking for pointers on where to start to get a package of mine added to Homebrew.

**Comment by SMillerDev** [comment]
> What’s the procedure for adding a new PyPI package? 

https://docs.brew.sh/Adding-Software-to-Homebrew

> [...] looking for pointers on where to start to get a package of mine added to Homebrew.

https://docs.brew.sh/Acceptable-Formulae#niche-or-self-submitted-stuff

**Comment by masaccio** [comment]
So the mappings in `pypi_formula_mappings.json` are not used to generate formulae, but instead I need to create one from scratch?

**Reply by SMillerDev** [reply, marked as answer]
Yes, the mappings are just to help create formula updates.
